wiki/Tournaments/AESS/2/en.md

```diff
@@ -0,0 +1,405 @@
+  2. [Kurokotei - Chronostasis (knowledgeking) [Extra]](https://osu.ppy.sh/beatmapsets/1247724#osu/2764088)
+  3. [Kotone - Underworld Utautai (captin1) [sub to kotone]](https://osu.ppy.sh/beatmapsets/1557296#osu/3181310)
+  4. [seatrus - MONONOKE (Realazy) [Orientalism]](https://osu.ppy.sh/beatmapsets/1280194#osu/2659353)
+  5. [goreshit - xenobeat (hammyMyRammy) [fumo]](https://osu.ppy.sh/beatmapsets/1569789#osu/3205607)
+- Hidden
+  1. [Akatsuki Records - KARMANATIONS (-Rik-) [Reincarnation]](https://osu.ppy.sh/beatmapsets/1283387#osu/2665294)
+  2. [REDALiCE - Taboo tears you up 2017 (Dada) [Vintage]](https://osu.ppy.sh/beatmapsets/1616043#osu/3299402)
+  3. [Gentle Stick X M2U - Ineffabilis (Bakugo-) [Skubi x Bakugo-'s Ineffable]](https://osu.ppy.sh/beatmapsets/1334202#osu/2989340)
+- HardRock
+  1. [Imperial Circus - L.G.C.-the Last Governed Connector- (AmatsuYzu) [Extra]](https://osu.ppy.sh/beatmapsets/788991#osu/1988250)
+  2. [ak+q - Vivid Theory (Cubby) [Future]](https://osu.ppy.sh/beatmapsets/1184160#osu/2468594)
+  3. [Mili - String Theocracy ([Arc]) [Theatrically Interesting]](https://osu.ppy.sh/beatmapsets/1192164#osu/2484169)
+- DoubleTime
+  1. [Akatsuki Records - WARNING x WARNING x WARNING (Xenon-) [Insane]](https://osu.ppy.sh/beatmapsets/860440#osu/1813044)
+  2. [Endorfin. - Raindrop Caffe Latte (ImpurePug) [Insane]](https://osu.ppy.sh/beatmapsets/1262556#osu/2624225)
+  3. [dj TAKA - Colors -sasakure.UK Futurelogic Remix- (wa_) [Hyper]](https://osu.ppy.sh/beatmapsets/317439#osu/1319822)
+- Tiebreaker
+  1. **[Emille's Moonlight Serenade - Kimi no Na wo Hibikasete (Senseabel) [a promise beyond the sky]](https://osu.ppy.sh/beatmapsets/938781#osu/1961046)**
+
+### Qualifiers
+
+- NoMod
+  1. [Wasuta - Saijoukyuu Paradox (xChillin) [Pupipapi PAPAPAPAPAN]](https://osu.ppy.sh/beatmapsets/1361574#osu/2817353)
+  2. [F-777 - Vortex (knowledgeking) [THREE DIMENSIONS]](https://osu.ppy.sh/beatmapsets/1543390#osu/3154777)
+  3. [KINEMA106 - IMPERIAL PARADE (Atalanta) [WORSHIP ME!!!]](https://osu.ppy.sh/beatmapsets/1569800#osu/3205635)
+  4. [Zekk - Re_Construct (Acylica) [captin's Extra(#4)]](https://osu.ppy.sh/beatmapsets/1318329#osu/3136135)
+- Hidden
+  1. [Fujikawa Chiai - Gomi no Hi (a_Blue) [Extra]](https://osu.ppy.sh/beatmapsets/1235121#osu/2729414)
+  2. [BAND-MAID - Spirit!! (Lasse) [Expert!!]](https://osu.ppy.sh/beatmapsets/1306578#osu/2709014)
+- HardRock
+  1. [BUD VIRGIN LOGIC - x Senritsu -Schlehit Melodie- (Mordred) [Lasse's Bud Light Extra]](https://osu.ppy.sh/beatmapsets/1569807#osu/3217086)
+  2. [ak+q - Axium Crisis (MrSergio) [Collapse]](https://osu.ppy.sh/beatmapsets/812551#osu/1704329)
+- DoubleTime
+  1. [S3RL - Candy (feat. Sara) (Nuvolina) [Nuvolina & Anto's Insane]](https://osu.ppy.sh/beatmapsets/829956#osu/1742403)
+  2. [Akiyama Uni - The Grimoire of Alice (Hollow Wings)[Insane]](https://osu.ppy.sh/beatmapsets/123839#osu/344619)
+
+
+## Match results
+
+### Grand Finals
+
+2022-07-23/2022-07-24
```

Comment:
did the match take place across two days?

have a look at [an example](https://osu.ppy.sh/wiki/en/Tournaments/OWC/2021#match-results) for the date formatting here